app/src/main/java/ai/elimu/appstore/receiver/InstallCompleteReceiver.java

```diff
@@ -21,11 +29,13 @@ public void setInstallCompleteCallback(@NonNull InstallCompleteCallback
     }
 
     /**
-     * A listener that listens to install completion event
+     * A listener that listens to install/uninstall completion event
      */
     public interface InstallCompleteCallback {
 
         void onInstallComplete(@NonNull String packageName);
 
+        void onUninstallComplete(@NonNull String packageName);
```

Comment:
Do we have functionality for uninstalling an application?

Reply:
No we have not yet had. The callback function is for resetting Download button state after uninstalling an app.

Reply:
Should not this be put in a separate receiver? It's confusing that the interface is called `InstallCompleteCallback` and contains code related to uninstallation.

Reply:
Ok. Then could you change then name of the interface?

Reply:
Ok. Then could you change the name of the interface?